ios/RNFrequency.m

```diff
@@ -10,6 +10,11 @@ @interface RNFrequency ()
 @implementation RNFrequency
 RCT_EXPORT_MODULE(Frequency);
 
++ (BOOL)requiresMainQueueSetup
+{
+   return NO;
```

Comment:
Do we know enough about `TGSineWaveToneGenerator`'s initialization process for it to safely initialize on a non-main thread? `YES` might be a safe fallback

Reply:
👍 Fixed and rebased the change in